src/services/positionService.ts

```diff
@@ -734,6 +768,37 @@ export default class PositionService {
     return this.providerService.sendTransactionWithGasLimit(tx);
   }
 
+  async withdrawSafe(position: Position) {
+    const currentNetwork = await this.providerService.getNetwork();
+    const wrappedProtocolToken = getWrappedProtocolToken(currentNetwork.chainId);
+    const toToUse = position.to.address === PROTOCOL_TOKEN_ADDRESS ? wrappedProtocolToken : position.to;
```

Comment:
I'm not familiar with the code, but let' say that a position has "USDC generating yield on Aave" as "to". Is `position.to` "USDC" or "USDC generating yield on Aave"?

Reply:
the .to is `USDC` and the underlying token is the `waUSDC`, I changed this to make displaying the view of the positions easier